src/ng/compile.js

```diff
@@ -1319,13 +1318,6 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
         // modify it.
         $compileNodes = jqLite($compileNodes);
       }
-      // We can not compile top level text elements since text nodes can be merged and we will
-      // not be able to attach scope data to them, so we will wrap them in <span>
```

Comment:
OOC, what has changed and this comment is no longer valid ?
Why did we need to wrap them in `<span>`, but not any more ?